### PR TITLE
update vault usage doc

### PR DESCRIPTION
**Pull request by cuttingedge1109**

### CODE REVIEW

**Comment by stakater-user** [comment]
@cuttingedge1109 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-84-baec8e5d`

File: content/sre/secrets/vault.md [hunk 1]
```diff
@@ -28,118 +28,37 @@ Tokens are created on demand with a specified expiry time and can be revoked at
...
+  Users can create/delete/update/read secrets on the `TENANT_NAME/*` path.
+
+Users can manage secrets via vault UI or vault CLI.
+## Using Vault UI
+Once the user is included in any tenants, he can access to the Vault UI using OIDC authentication.
+**Step**
+* Access https://stakater-vault-openshift-stakater-vault.CLUSTER_DOMAIN
+* Select `OIDC` method on `Sing in to Vault` page.
+* Keep `Role` as default.
+* Click `Sign in with OIDC Provider` and sign in to the proper IdP.
```
Comment (rasheedamir) [inline content/sre/secrets/vault.md:52]: @cuttingedge1109 plz add a screen shot

File: content/sre/secrets/vault.md [hunk 2]
```diff
@@ -28,118 +28,37 @@ Tokens are created on demand with a specified expiry time and can be revoked at
...
+Users can do all actions on the path `TENANT_NAME/*`.
+
+- Enable/disable any kinds of secret engines
+
+- Create/update/get/list/delete secrets
+
+## Using Vault CLI
+To use vault CLI, the token is required. Users can get/renew/revoke the token on the UI. (Click the user account Avatar.)
+```bash
+vault login token=${TOKEN}
```
Comment (rasheedamir) [inline content/sre/secrets/vault.md:72]: @cuttingedge1109 plz add a screen shot and recommend user to use the vault terminal available in vault UI so, they don't have to install something locally

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-84-aa50ca27`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-84-0db930eb`

**Review by rasheedamir — APPROVED** [review]

**Comment by stakater-user** [comment]
@cuttingedge1109 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-84-e05f66d9`

**Comment by stakater-user** [comment]
@cuttingedge1109 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:v0.0.41`

**Pull request merged**